BUILD.gn

```diff
@@ -72,7 +72,10 @@ group("flutter") {
     ]
 
     if (is_win) {
-      public_deps += [ "$flutter_root/shell/platform/windows/client_wrapper:client_wrapper_windows_unittests" ]
+      public_deps += [
+        "$flutter_root/shell/platform/windows/client_wrapper:client_wrapper_windows_unittests",
```

Comment:
@stuartmorgan do we have presumbit task to run the tests?

Reply:
It looks like we don't run the client wrapper for windows, is this intentional?

Reply:
No, it just never got set up.